internal/datacoord/meta.go

```diff
@@ -993,6 +1011,10 @@ func (m *meta) PrepareCompleteCompactionMutation(plan *datapb.CompactionPlan,
 	for _, cl := range compactionLogs {
 		if segment := m.segments.GetSegment(cl.GetSegmentID()); segment != nil {
 			cloned := segment.Clone()
+			err := binlog.DecompressBinLog(storage.DeleteBinlog, cloned.GetCollectionID(), cloned.GetPartitionID(), cloned.GetID(), cloned.GetDeltalogs())
```

Comment:
Should we decompress here?
Seems that updateDeltalogs is using logID?is there anywhere we use logpath here

Reply:
yes, this is the only place where using logpath in compaction, logpath is needed when writing new objects into kv.